code-of-conduct/template-code-of-conduct.md

```diff
@@ -0,0 +1,144 @@
+
+
+## Behavior that will not be tolerated
+
+The following behaviors are unacceptable within our community, whether they occur online, offline, privately, or publicly:
+
+- Violent threats or language directed against another person.
+- Sexist, racist, or otherwise discriminatory jokes and language.
+- Deliberate intimidation, stalking, or following (online or in person).
+- Personal insults, especially those related to gender, sexual orientation, race, religion, or disability. This includes misgendering, name calling, and mockery.
+- Trolling (posting controversial comments in order to provoke others), insulting, or making derogatory comments.
+- The use of sexualized language, jokes, or imagery, and sexual attention, inappropriate physical contact, or sexual advances of any kind.
+- Bullying, tone-policing (attacking a person’s tone rather than the content of their message), or repeatedly dominating a topic of conversation (such as regularly talking over another person or not inviting discussion from others where appropriate).
+- Publishing or threatening to publish others’ private information without their explicit permission. Private information includes physical addresses, email addresses, and emails or other communications sent privately or non-publicly.
+- Deliberate “outing” of any private aspect of a person’s identity without their consent except as necessary to protect vulnerable people from intentional abuse. This includes sharing personally identifying information (“doxing”).
+- Excessive or unnecessary profanity.
+- Deliberate misgendering.
+- Knowingly making harmful false claims about a person.
+- Pushing a person to drink alcohol when they don’t want to drink, or deceiving someone into drinking alcohol.
+- Marketing to our community members either individually or collectively without their express approval. Solicitations on behalf of your business should be restricted to designated channels or areas. For more information about appropriate ways to market your business to the community, ask one of the {Project name} {link to your community moderators document [community moderators](../our-team/template-our_team.md)}.
+- Repeatedly communicating with a community member who has asked you to leave them alone.
+- Sustained disruption of community events, including meetings and presentations.
+- Recording or photography at community meetings and events without explicit permission.
+- Advocating for, or encouraging, any of the above behavior.
+- Other conduct which could reasonably be considered inappropriate in a professional setting.
+
+
+## Consequences of unacceptable behavior
+
+Unacceptable behavior from any {Project name} community member, contributor, sponsors, or leaders will not be tolerated.

```

Comment:
suggestion: Can we remove {Project name}? I just tried replacing it with an actual project name and it didn't sound right to me.

Reply:
Hmmm... It doesn't sound right to you? It sounds right to me:

"Unacceptable behavior from any Salt Project community member..."

"Unacceptable behavior from any Good Docs Project community member..."

Can you tell me more about what doesn't sound right to you?